src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -96,6 +96,8 @@ public boolean compile(String cp) throws IOException, InterruptedException {
         FileOps.rmdir(paths.getTmpPath());
         String tmpDir = paths.getTmpPath().toFile().getAbsolutePath();
         String mainClassName = projectConfiguration.getMainClassName();
+        List<String> compilerArgs = projectConfiguration.getCompilerArgs();
+
```

Comment:
This is not necessary, add directly the compiler args to the process builder below